cheroot/test/test_server.py

```diff
@@ -186,8 +186,14 @@ def test_peercreds_unix_sock(http_server, unix_sock_file):
     assert testclient.get(PEERCRED_IDS_URI) == expected_peercreds
     assert 'RuntimeError' in testclient.get(PEERCRED_TEXTS_URI)
 
-    httpserver.peercreds_resolve_enabled = True
-    import grp
-    expected_textcreds = os.getlogin(), grp.getgrgid(os.getgid()).gr_name
-    expected_textcreds = '!'.join(map(str, expected_textcreds))
-    assert testclient.get(PEERCRED_TEXTS_URI) == expected_textcreds
+    try:
+        import grp
+    except ImportError:
```

Comment:
We should use `skipif` marker on the test instead of having a guessing logic inside of it since it's unrelated. You can create a named marker like `non_windows_sock_test` is.

Reply:
I'm not familiar with pytest. The whole `test_peercreds_unix_sock` should be skipped, when `grp` is not importable? I don't think so. But this is your decision.

Reply:
Well, it should probably be separated into its own test. Actually, currently, this one is completely disabled, because the test client is unable to HTTP over non-IP connection (yet).

As for `pytest`, you can have a decorator like `@pytest.mark.skipif(<condition_here>, reason="some explanation"`, which would conditionally skip the test.

Reply:
I think you should change it to your own.

Reply:
ok